README.md

```diff
@@ -1,68 +1,64 @@
-9wm
-====
+# 9wm
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½, also known as rio.
-It provides a very simple and clean user interface.
-It is click-to-type.
-It uses the X11 font system (which, unfortunately, means no Unicode support).
+9wm is an X11 window manager. It's inspired by the Plan 9 window manager 8½, also known as [rio].
 
-9wm does not provide virtual desktops, customization, key bindings,
-EWMH support, or compositing.
-It does not allocate any colors (if you disable COLOR),
-which will be great news if you are stuck in 1993.
+- Simple and clean user interface. It is click-to-type.
+- It uses the X11 font system (which, unfortunately, means no Unicode support).
+- No virtual desktops, customization, key bindings, EWMH support, or compositing.
+- It does not allocate any colors (if you disable COLOR), which will be great news if you are stuck in 1993.
```

Comment:
Please revert to the original text. Your bulleted list appears with no introduction of why we are listing things, you have incomplete sentences, and you have multiple items in the first item.